docs/ERC-725.md

```diff
@@ -39,22 +39,36 @@ And the event:
 
 ### ERC725X
 
-ERC165 identifier: `0x44c028fe`
+ERC165 identifier: `0x570ef073`
+
+**Note:** The `execute()` function use function overloading, therefore it is better to reference them by the given function signature as follows:
+
+```js
+// web3.js example
+myContract.methods['execute(uint256,address,uint256,bytes)'](OPERATION_CALL, target.address, 2WEI, "0x").send();
+
+myContract.methods['execute(uint256[],address[],uint256[],bytes[])']([OPERATION_CALL, OPERATION_CREATE], [target.address, ZERO_ADDRESS], [2WEI, 0WEI], ["0x", CONTRACT_BYTECODE]).send()
+
+// or
+myContract.methods['0x44c028fe'](OPERATION_CALL, target.address, 2WEI, "0x").send();
```

Comment:
Could be please add all examples. Add the function signature for both functions, with a comment after which function it is pls